jobs/config.json

```diff
@@ -7797,6 +7797,23 @@
       "sig-aws"
     ]
   },
+  "ci-kubernetes-e2e-kops-gce": {
+    "args": [
+      "--aws",
+      "--cluster=e2e-kops-gce.k8s.local",
+      "--env-file=jobs/platform/kops_gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-kops-gce.env",
+      "--extract=ci/latest",
+      "--ginkgo-parallel",
+      "--provider=gce",
+      "--test_args=--ginkgo.flakeAttempts=2 --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]|\\[HPA\\]|Dashboard|Services.*functioning.*NodePort",
```

Comment:
This is the same test args on `ci-kubernetes-e2e-kops-aws` (separately, I am removing the exceptions e.g. for NodePort)